app/views/Project/Tagging/Export/NewAssessmentExport/index.tsx

```diff
@@ -93,24 +82,32 @@ function NewAssessmentExport(props: Props) {
 
     const {
         value: sourcesFilterValue,
-        setFieldValue: setSourcesFilterValue,
+        setFieldValue: setSourcesFilterFieldValue,
+        setValue: setSourcesFilterValue,
         resetValue: clearSourcesFilterValue,
         pristine,
         validate,
         setError,
         setPristine,
     } = useFilterState();
 
+    useEffect(() => {
+        if (state?.filters) {
+            setSourcesFilterValue(transformRawFiltersToFormValues(state?.filters));
+        }
+    }, [setSourcesFilterValue, state?.filters]);
```

Comment:
Does this render only on the first render?

Reply:
yes